brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/value/ResStringValue.java

```diff
@@ -26,13 +26,10 @@
  * @author Ryszard Wiśniewski <brut.alll@gmail.com>
  */
 public class ResStringValue extends ResScalarValue {
+    public ResStringValue(String value, int rawValue) { this(value, rawValue, "string"); }
```

Comment:
I see you added the `rawValue` here, but the formatting is one line now. Restore this w/ the added changes to the prior formatting if you could.

ie

```
public foo() {
}
```